its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -670,21 +670,24 @@ void testEsprojVueWithBackend() throws IOException {
         .extracting(Issue::getRule, Issue::getComponent)
         .contains(expectedIssues.toArray(new Tuple[]{}));
     }
-    assertThat(TestUtils.getMeasureAsInteger(localProjectKey, "lines", ORCHESTRATOR)).isIn(307, 2115, 2120, 18681);
-    assertThat(TestUtils.getMeasureAsInteger(localProjectKey, "ncloc", ORCHESTRATOR)).isIn(243, 2049, 2054, 14028);
-    assertThat(TestUtils.getMeasureAsInteger(localProjectKey, "files", ORCHESTRATOR)).isIn(10, 13, 213);
+    // Different expected values are for different SQ and MsBuild versions and local run
+    assertThat(TestUtils.getMeasureAsInteger(projectKey, "lines", ORCHESTRATOR)).isGreaterThan(300);
+    assertThat(TestUtils.getMeasureAsInteger(projectKey, "ncloc", ORCHESTRATOR)).isGreaterThan(200);
+    assertThat(TestUtils.getMeasureAsInteger(projectKey, "files", ORCHESTRATOR)).isGreaterThanOrEqualTo(10);
```

Comment:
😄 This is the third version now:
https://github.com/SonarSource/sonar-scanner-msbuild/pull/2334#discussion_r1973732564

Reply:
I had way too many new values :)